reference/5.0/Microsoft.PowerShell.Archive/Compress-Archive.md

```diff
@@ -16,98 +17,130 @@ Creates an archive, or zipped file, from specified files and folders.
+compressed by using the compression algorithm specified by the CompressionLevel parameter.
+
+Because `Compress-Archive` relies upon the Microsoft .NET API `System.IO.Compression.ZipArchive` to
+compress files. Therefor, the maximum file size that is 2 GB. This is a limitation of the underlying
+API.
 
 ## EXAMPLES
 
 ### Example 1: Create an archive file
+
+This command creates a new archive file, Draft.zip, by compressing two files, Draftdoc.docx and
+diagram2.vsd, specified by the **Path** parameter. The compression level specified for this
+operation is Optimal.
+
+```powershell
+Compress-Archive -Path C:\Reference\Draftdoc.docx, C:\Reference\Images\diagram2.vsd -CompressionLevel Optimal -DestinationPath C:\Archives\Draft.Zip
+```
+
+### Example 2: Create an archive file (using LiteralPath)
+
+This command creates a new archive file, Draft.zip, by compressing two files, "Draft doc.docx" and
+"Diagram [2].vsd", specified by the **LiteralPath** parameter. The compression level specified for this
+operation is Optimal.
+
 ```powershell
-Compress-Archive -LiteralPath C:\Reference\Draftdoc.docx, C:\Reference\Images\diagram2.vsd -CompressionLevel Optimal -DestinationPath C:\Archives\Draft.Zip
+Compress-Archive -LiteralPath 'C:\Reference\Draft Doc.docx', 'C:\Reference\Images\Diagram [2].vsd'  -CompressionLevel Optimal -DestinationPath C:\Archives\Draft.Zip
 ```
 
-This command creates a new archive file, Draft.zip, by compressing two files, Draftdoc.docx and diagram2.vsd, specified by the **LiteralPath** parameter.
-The compression level specified for this operation is Optimal.
+### Example 3: Create an archive with wildcard characters
+
+This command creates a new archive file, Draft.zip, in the C:\Archives folder. The new archive file
+contains every file in the C:\Reference folder, because a wildcard character was used in place of
+specific file names in the **Path** parameter.
 
-### Example 2: Create an archive with wildcard characters
 ```powershell
 Compress-Archive -Path C:\Reference\* -CompressionLevel Fastest -DestinationPath C:\Archives\Draft
 ```
 
-This command creates a new archive file, Draft.zip, in the C:\Archives folder.
-Note that though the file name extension .zip was not added to the value of the **DestinationPath** parameter, Windows PowerShell appends this to the specified archive file name automatically.
-The new archive file contains every file in the C:\Reference folder, because a wildcard character was used in place of specific file names in the **Path** parameter.
-The specified compression level is Fastest, which might result in a larger output file, but compresses a large number of files faster.
+Notice that the file name extension .zip was not added to the value of the **DestinationPath**
+parameter. PowerShell appends the .zip extension to the file name automatically. The specified
+compression level is Fastest, which might result in a larger output file, but compresses a large
```

Comment:
Is Fastest a parameter value? Since it is, does it need some formatting?

Reply:
fixed